u2-routing-controllers/features/tempController.js

```diff
@@ -0,0 +1,20 @@
+const getTemp = (req, res) => {
+  const u1 = req.params.u1.toUpperCase()
+  const u2 = req.params.u2.toUpperCase()
+  const t1 = req.params.t1
+
+  let t2
+  if (u1 === u2) {
+    t2 = t1
+  } else if (u1 === 'C') {
+    t2 = t1 * 9 / 5 + 32
+  } else if (u1 === 'F') {
+    t2 = (t1 - 32) * 5 / 9
+  }
+
+  res.send(`${parseFloat(t1).toFixed(1)} ${u1} = ${parseFloat(t2).toFixed(1)} ${u2}`)
```

Comment:
หากคุณแยกวิเคราะห์ `t1` ในบรรทัดที่ 4 คุณไม่จำเป็นต้อง parseFloat สำหรับ `t1` หรือ `t2` ที่นี่ (ทั้งคู่ลอยอยู่แล้ว)
If you parse `t1` on line 4, then you don't need parseFloat for either `t1` or `t2` here (they are both already floats).